src/hooks/useLocale.jsx

```diff
@@ -0,0 +1,22 @@
+import Spanish from "../lang/es-US.json";
+import English from "../lang/en-US.json";
+
+// detect the language preference of the user's browser
+const useLocale = () => {
+  const locale = navigator.language;
+
+  let messages;
+  switch (locale) {
+    case "en-US":
+      messages = English;
+      break;
+    case "es-US":
+      messages = Spanish;
+      break;
+    default:
+      messages = English;
+  }
```

Comment:
❓ Can we use `useEffect` here?

Reply:
Addressed:  b00c88249d441a7b6f25d3d112c8f5d08b4bd7bd